applications/Contract_wizard.md

```diff
@@ -0,0 +1,171 @@
+- https://github.com/protofire/omen-exchange
+- https://github.com/protofire/kyber-pool-master-proxy-contract
+​
+GitHub accounts of team members.
+- https://github.com/alongoni <Agustin Langoni>
+- https://github.com/henrypalacios <Henry Palacios>
+- https://github.com/0xLucca <Luca Auet>
+- https://github.com/GabrielCamba <Gabriel Gonzalez>
+​
+### Team LinkedIn Profiles
+​
+- https://www.linkedin.com/in/diego-torres-borda-94b70912/
+- https://www.linkedin.com/in/agustinlongoni/
+- https://www.linkedin.com/in/henrydpalacios/
+- https://www.linkedin.com/in/lucaauet/
+- https://www.linkedin.com/in/gabriel-nicolas-gonzalez/
+​
+## Development Status :open_book:
+​
+We have successfully completed the previous grant, and we can confidently affirm that our platform, [contractwizard.xyz](https://www.contractwizard.xyz/), is now a user-friendly and highly useful tool. One of the key features of our platform is that it simplifies the deployment of standard smart contracts within the Dotsama ecosystem. With our platform, users no longer have to navigate through complex processes to deploy their contracts. Our streamlined approach means that anyone can deploy a smart contract without requiring advanced technical knowledge, saving time and resources. We are incredibly grateful for the support we have received so far, and we are committed to ensuring that our platform remains a top choice for deploying smart contracts within the Dotsama ecosystem. 
+​
+## Development Roadmap :nut_and_bolt:
+​
+Described in project details.
+​
+### Overview
+​
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):** 2.5 
+- **Total Costs:** 52500 U$D
+​
+## Milestones
+​
+### Milestone 1 — Functionality Implementation and Educational Content
+​
+- **Estimated duration:** 3 months
+- **FTE:** 2.5
+- **Costs:** 52500
+  
+​
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can use the new features. |
+| **0c.** | Testing and Testing Guide | The code will have unit-test coverage to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile that allows users to easily run the application. |
+| 1. | Interaction with deployed contracts | The functionallity based on [step 1](#step1) of the Project Details section. |
+| 2. | Interaction with custom smart contracts | The functionality based on [step 2](#step2) of the Project Details section. |
+| 3. | Shareable Contract Interaction Links | The functionality corresponds to what's described on [step 3](#step3) of the Project Details section. |
+​| 4. | Support for multiple chains | The functionality to be implemented corresponds to [step 4](#step4) of the Project Details section. |
```

Comment:
```suggestion
| 3. | Shareable Contract Interaction Links | The functionality corresponds to what's described on [step 3](#step3) of the Project Details section. |
| 4. | Support for multiple chains | The functionality to be implemented corresponds to [step 4](#step4) of the Project Details section. |
```

There was a zero-width space in the last line, messing with the markdown. Should be fixed with this.

Reply:
Done! Thanks!